src/vehicle.cpp

```diff
@@ -2455,7 +2464,7 @@ item_group::ItemList vehicle_part::pieces_for_broken_part() const
 }
 
 std::vector<int> vehicle::parts_at_relative( const point &dp,
-        const bool use_cache ) const
+        const bool use_cache, bool include_fake ) const
 {
     if( !use_cache ) {
         std::vector<int> res;
```

Comment:
This branch of the `if` doesn't check `include_fake`. It will always return a vector with fake parts, even when `include_fake` is false.